addons/stock_account/views/stock_valuation_layer_views.xml

```diff
@@ -50,17 +50,21 @@
                             class="btn-primary ms-1"
                             display="always"/>
                 </header>
+                <field name="account_move_id" invisible="1"/>
                 <field name="create_date" string="Date" />
                 <field name="reference"/>
-                <button name="action_open_layer" icon="oi-arrow-right" title="Open Valuation Layer" type="object"/>
+                <field name="account_move_id" optional="hide"/>
```

Comment:
@yhu-odoo, we require this field, "account_move_id," to be optional hidden. This is because, in the specifications of another task, I have instructed that this field should be optional hidden. You can find the details of this task here:https://www.odoo.com/web#id=3388131&menu_id=4720&cids=2&action=333&active_id=966&model=project.task&view_type=form